examples/configs/classification_cifar10_mlflowx.yaml

```diff
@@ -94,15 +94,19 @@ seed_params:
 # For MLFlowLogger and MLFlowLoggerX need to define tracking_uri in params - address of local or remote tracking server,
 # if it isn't define it will be gotten from MLFLOW_TRACKING_URI - environment variable
 logger:
-  log_dir: '${oc.env:HOME}/.cache/torchok/logs/'
   experiment_name: cifar10
+  log_dir: '${oc.env:HOME}/${logger.experiment_name}/logs'
   name: MLFlowLoggerX
+  timestamp: '${now:%Y-%m-%d}/${now:%H-%M-%S}'
   params:
-    artifact_location: 'file://${oc.env:HOME}/.cache/torchok/logs/artifacts/'
+    run_name: resnet18
+    tracking_uri: 'mysql+pymysql://${oc.env:DB_USER}:${oc.env:DB_PASSWORD}@${oc.env:DB_HOST}/mlflow'
+    save_dir: '${logger.log_dir}/${logger.params.run_name}/${logger.timestamp}'
```

Comment:
Why `save_dir` and `log_dir` are different?

Reply:
Remove save_dir as it is not used

Reply:
I don't see changes